tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml

```diff
@@ -125,7 +125,7 @@ jobs:
         --test-series "master" \
         --locale "en_US" \
         --build-dir $(Build.BinariesDirectory)/app_center_test_mobile/ios_package_test/DerivedData/Build/Products/Debug-iphoneos \
-        --token $(app_center_api_token)
+        --token $(app_center_api_token_2)
```

Comment:
can we just replace the old value of $(app_center_api_token)?

Reply:
Sure, updated the old value. will leave this change open as no need for pipeline change required.